external/js/cothority/dist/bundle.min.js

```diff
@@ -0,0 +1 @@
u,a,e[c+15],14,-660478335),l,u,e[c+4],20,-405537848),f=n(f,l=n(l,u=n(u,a,f,l,e[c+9],5,568446438),a,f,e[c+14],9,-1019803690),u,a,e[c+3],14,-187363961),l,u,e[c+8],20,1163531501),f=n(f,l=n(l,u=n(u,a,f,l,e[c+13],5,-1444681467),a,f,e[c+2],9,-51403784),u,a,e[c+7],14,1735328473),l,u,e[c+12],20,-1926607734),f=o(f,l=o(l,u=o(u,a,f,l,e[c+5],4,-378558),a,f,e[c+8],11,-2022574463),u,a,e[c+11],16,1839030562),l,u,e[c+14],23,-35309556),f=o(f,l=o(l,u=o(u,a,f,l,e[c+1],4,-1530992060),a,f,e[c+4],11,1272893353),u,a,e[c+7],16,-155497632),l,u,e[c+10],23,-1094730640),f=o(f,l=o(l,u=o(u,a,f,l,e[c+13],4,681279174),a,f,e[c+0],11,-358537222),u,a,e[c+3],16,-722521979),l,u,e[c+6],23,76029189),f=o(f,l=o(l,u=o(u,a,f,l,e[c+9],4,-640364487),a,f,e[c+12],11,-421815835),u,a,e[c+15],16,530742520),l,u,e[c+2],23,-995338651),f=s(f,l=s(l,u=s(u,a,f,l,e[c+0],6,-198630844),a,f,e[c+7],10,1126891415),u,a,e[c+14],15,-1416354905),l,u,e[c+5],21,-57434055),f=s(f,l=s(l,u=s(u,a,f,l,e[c+12],6,1700485571),a,f,e[c+3],10,-1894986606),u,a,e[c+10],15,-1051523),l,u,e[c+1],21,-2054922799),f=s(f,l=s(l,u=s(u,a,f,l,e[c+8],6,1873313359),a,f,e[c+15],10,-30611744),u,a,e[c+6],15,-1560198380),l,u,e[c+13],21,1309151649),f=s(f,l=s(l,u=s(u,a,f,l,e[c+4],6,-145523070),a,f,e[c+11],10,-1120210379),u,a,e[c+2],15,718787259),l,u,e[c+9],21,-343485551),u=h(u,p),a=h(a,y),f=h(f,v),l=h(l,g)}return[u,a,f,l]}(s(e,{ibits:8,obits:32,obigendian:!1}),8*e.length),{ibits:32,ibigendian:!1})},g=0,m=0,b=function(){if(1===arguments.length&&"string"==typeof arguments[0])this.parse.apply(this,arguments);else if(arguments.length>=1&&"number"==typeof arguments[0])this.make.apply(this,arguments);else{if(arguments.length>=1)throw new Error("UUID: constructor: invalid arguments");for(var e=0;e<16;e++)this[e]=0}};return(b.prototype="undefined"!=typeof Uint8Array?new Uint8Array(16):i?new i(16):new Array(16)).constructor=b,b.prototype.make=function(t){var r;if(1===t){var i=(new Date).getTime();i!==g?m=0:m++,g=i;var n,o=l(i);!function(e,t){for(var r=0,i=0;i<8;i++)r+=e[i]*t,e[i]=Math.floor(r%256),r=Math.floor(r/256)}(o,1e4),c(o,a(1,178,29,210,19,129,64,0)),m>0&&c(o,l(m)),n=p(o,8),this[3]=255&n,n=p(o,8),this[2]=255&n,n=p(o,8),this[1]=255&n,n=p(o,8),this[0]=255&n,n=p(o,8),this[5]=255&n,n=p(o,8),this[4]=255&n,n=p(o,8),this[7]=255&n,n=p(o,8),this[6]=15&n;var s=e(2,255);this[8]=s[0],this[9]=s[1];var u=e(6,255);for(u[0]|=1,u[0]|=2,r=0;r<6;r++)this[10+r]=u[r]}else if(4===t){var f=e(16,255);for(r=0;r<16;r++)this[r]=f[r]}else{if(3!==t&&5!==t)throw new Error("UUID: make: invalid version");var h="",d="object"==typeof arguments[1]&&arguments[1]instanceof b?arguments[1]:(new b).parse(arguments[1]);for(r=0;r<16;r++)h+=String.fromCharCode(d[r]);h+=arguments[2];var w=3===t?v(h):y(h);for(r=0;r<16;r++)this[r]=w.charCodeAt(r)}return this[6]&=15,this[6]|=t<<4,this[8]&=63,this[8]|=128,this},b.prototype.format=function(e){var r,i;return"z85"===e?r=function(e,t){if(t%4!=0)throw new Error("z85_encode: invalid input length (multiple of 4 expected)");for(var r="",i=0,o=0;i<t;)if(
```

Comment:
Why are minified files checked in? To me, these are like binaries, they don't belong in Git.

Reply:
I see your point. Mine was to be able to already offer an online working version of the script directly (just `<script src="https://github.com/dedis/cothority/external/js/kyber"></script>`). But that may not be the best option indeed. We need to think of a strategy to be able to deliver online scripts as these. 
I'll remove it.